src/coreclr/jit/valuenum.cpp

```diff
@@ -106,7 +106,7 @@ struct DoubleTraits
 template <typename TFp, typename TFpTraits>
 TFp FpAdd(TFp value1, TFp value2)
 {
-#if defined(TARGET_ARMARCH) || defined(TARGET_LOONGARCH64)
```

Comment:
cc @dotnet/jit-contrib

Reply:
Why is any change to valuenum.cpp needed here for the cbor fix?

This seems like an independent change related to constant folding which should be in it's own PR.

Reply:
OK, #93098